bin/hoc-2015-survey/test-params-organizers.csv

```diff
@@ -0,0 +1,5 @@
+email,name,international,domestic,no_posters
```

Comment:
How do we handle someone with a comma in their name? Will it get escaped properly?